todo_app/data/session_items.py

```diff
@@ -1,69 +1,108 @@
 from flask import session
+import requests, os
+from dotenv import load_dotenv
+from flask.helpers import url_for
+from todo_app import item
```

Comment:
Could do `from todo_app.item import Item` if you don't want to have to do `item.Item` to use the class